### PR TITLE
Update to ruby version 2.5.1 

### PR DESCRIPTION
**Pull request by jamuc**
What the title says 🆙

### CODE REVIEW

**Comment by jamuc** [comment]
@BasThomas `Status: Downloaded newer image for circleci/ruby:2.5.1` 🎂

**Comment by jamuc** [comment]
@BasThomas To be honest, I've never really worked with change logs. I assume it's next up? Or should I create an entry for the next version?

**Comment by BasThomas** [comment]
> @BasThomas To be honest, I've never really worked with change logs. I assume it's next up? Or should I create an entry for the next version?

I guess we can add some automation to this, but yeah, I'd add it to next up. Then when a new release goes out we can make the "Next up" the new version and add a new "Next up" section.

File: Changelog.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Next up
+ - Update crep to run with ruby version 2.5.1
```
Comment (BasThomas) [inline Changelog.md]: I suppose we should add it under the `Miscellaneous` section?

**Pull request merged**